test/core/both/test-bitswap.js

```diff
@@ -22,7 +22,7 @@ function makeBlock (cb) {
   return cb(null, new Block(`IPFS is awesome ${Math.random()}`))
 }
 
-describe('bitswap', () => {
+describe.skip('bitswap', () => {
```

Comment:
Is this being skipped just for the sake of testing faster? Well, let's make sure every test passes before the merge, remove the `skip`